AlpsSDK/ContextManager.swift

```diff
@@ -458,59 +497,53 @@ class ContextManager: NSObject, CLLocationManagerDelegate {
                    
                     if gap > 5 * 60 * 1000 {
                         t.removeValue(forKey: id)
-                        for i in 0...3 {
-                            switch i{
-                            case 0:
+                        for proximity in CLProximity.allValues {
```

Comment:
it seems like a lot of fors and switches.

Maybe we can come up with some nice functional code that would make it much shorter (better to maintain)

Reply:
👍  I agree !